test/e2e/wrappers.go

```diff
@@ -36,61 +32,6 @@ import (
 
 var k = k8s.KubeCtl{}
 
-func UpdateConfigMapWrapper(oldConfigMap *v1.ConfigMap, schedPolicy string, annotation string) {
```

Comment:
those two methods were introduced by https://github.com/apache/yunikorn-k8shim/commit/55e8148863a00b28be243f6e9ad38787726e065f. And they are similar to `yunikorn. UpdateConfigMapWrapper` and `yunikorn.RestoreConfigMapWrapper`. Was the error caused by fat finger or it intended to do something difference?

Reply:
Hi, @manirajv06, I noticed that this parameter is not being used, so I am starting the deletion process. 
However, I speculate that this design might involve assigning an annotation as a unique identifier during the UpdateConfigMapWrapper operation, and then using this annotation to retrieve it during the use of RestoreConfigMapWrapper.

This design could be aimed at preventing multiple tests from simultaneously using UpdateConfigMapWrapper, but I'm not entirely certain.

I guess you accidentally forgot it and then leave the annotation parameter here. Could you help me confirm that?

Reply:
> those two methods were introduced by https://github.com/apache/yunikorn-k8shim/commit/55e8148863a00b28be243f6e9ad38787726e065f. And they are similar to yunikorn. UpdateConfigMapWrapper and yunikorn.RestoreConfigMapWrapper. Was the error caused by fat finger or it intended to do something difference?

gentle ping @wilfred-s, I find myself uncertain here and would highly value your insights. Could you please share your thoughts?

Reply:
I'd like to ship this PR, so @amikai could you please file a follow-up PR to trace those similar functions? also this PR does NOT remove those similar functions

WDYT?

Reply:
Sounds good to me. I will create another PR to remove the unused variable, and keep these "similar functions" to this PR for discussion.

Reply:
Sounds good to me. I will create another PR to remove the unused variable, and keep these “similar functions” to this PR for discussion